sda-commons-shared-otel-instrumentation/README.md

```diff
@@ -0,0 +1,82 @@
+# SDA Commons Shared OpenTelemetry Instrumentation
+
+This module is responsible for loading the OpenTelemetry [Java agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation) dynamically to the current application.
+The agent provides automatic instrumentation for various popular libraries and frameworks [full List](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
+The agent can be configured to export the captured telemetry data produced by these libraries in different formats.
+
+# Docs
+An extensive documentation can be found in [OpenTelemetry Java documentation](https://opentelemetry.io/docs/instrumentation/java/). 
+
+# Usage
+
+This bundle requires the [openTelemetry agent](https://mvnrepository.com/artifact/io.opentelemetry.javaagent/opentelemetry-javaagent) to be added as a runtime dependency to the application.
+Sda-commons already provides an extension of the agent [sda-commons-shared-otel-agent](../sda-commons-shared-otel-agent) that can be used.
+
+```groovy
+  api project(':sda-commons-shared-otel-instrumentation')
+  runtimeOnly project(':sda-commons-shared-otel-agent')
+```
+
+Then the bundle is ready to be added to the application.
+
+```java
+public class MyApp extends Application<Configuration> {
+  @Override
+  public void run(Configuration configuration, Environment environment) {}
+
+  @Override
+  public void initialize(Bootstrap<Configuration> bootstrap) {
+    // ... other bundles  
+    bootstrap.addBundle(InstrumentationBundle.builder().build());
+  }
+}
+```
+
+If the application is already using the [starter bundle](../sda-commons-server-starter), no changes are needed.
```

Comment:
The bundle you're referencing is deprecated. Please refer to [sda-commons-starter](https://github.com/SDA-SE/sda-dropwizard-commons/tree/master/sda-commons-starter).